banners/desktop/C24_WMDE_Desktop_DE_17/components/BannerVar.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+	<div class="wmde-banner-wrapper" :class="contentState">
+		<MainBanner
+			@form-interaction="$emit( 'bannerContentChanged' )"
+			v-if="contentState === ContentStates.Main"
+			:bannerState="bannerState"
+		>
+			<template #close-button>
+				<div class="wmde-banner-minimised-button-group">
```

Comment:
I find this class name prefix `wmde-banner-minimised` confusing, because those elements don't belong to the minimised banner. It's convenient to style in the `MinimisedBanner.scss`, but I'd rather extract this into its own component with its own state and give it better class names